RFC_PROCESS.md

```diff
@@ -0,0 +1,39 @@
+# RFC Process
+
+## What's an RFC?
+
+The name itself is a reference to the IETF's Request For Comments process, and basically involves a document or series of documents which are drafted, reviewed, and eventually ratified (approved) by the Ayo.js community through discussion among those interested.
+
+An RFC can extend, modify, or alter any part of the Ayo.js project's code or documents, whether or not they've been previously documented. This includes proposing new features, proposing major breaking changes, changes to the Code of Conduct and other community policies, changes to the governance structure of the project, and anything else that affects a significant part of the community. It can also propose entirely new policies and community agreements.
+
+## How do I create an RFC?
+
+* Go to https://github.com/ayojs/ayo
```

Comment:
i'm against on having rfc's in the main repo, as it's already a collossal monolith. can we maybe move them to a separate repo, like rust does? i think it'd also help separate community rfc's away from the main code repo

Reply:
I'd like to advocate for keeping things in this repo for now, because there's relatively little volume and I think it'd be good to bootstrap an RFC process by putting it front-and-center. I would hate it if it ended up being seen like a dumping ground for unwanted ideas.